### PR TITLE
General testing for all legacy scripts

### PR DESCRIPTION
**Pull request by gabknight**
Added a test that calls all scilpy 1.6.0 scripts (170 scripts!). This replaces all others `scilpy/legacy/tests/test*.py` files.

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-838/2/

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-838/3/

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-838/4/

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-838/6/

**Comment by gabknight** [comment]
Script `scil_fix_dsi_studio_trk.py` is missing. Was this renamed and is this missing a legacy script or was it a dev script that got removed?

**Comment by gabknight** [comment]
It was renamed before we have the legacy folder - https://github.com/scilus/scilpy/commit/d831edb485f95f28814e72ac759e16271b737f7c

I'll add the legacy script.

**Review by EmmaRenauld — APPROVED** [review]
Thanks!

File: scripts/tests/test_tractogram_fix_trk.py [hunk 1]
```diff
@@ -3,5 +3,5 @@
 
 
 def test_help_option(script_runner):
-    ret = script_runner.run('scil_convert_tractogram.py', '--help')
+    ret = script_runner.run('scil_tractogram_fix_trk.py', '--help')
     assert ret.success
```
Comment (EmmaRenauld) [inline scripts/tests/test_tractogram_fix_trk.py:7]: In version 1.6., there was no more testing than the help?

Comment (arnaudbore) [reply scripts/tests/test_tractogram_fix_trk.py:7]: It's complicated to find example of bad trk :(  I will open an issue

**Pull request merged**